README.md

```diff
@@ -1,36 +1,71 @@
+# [Sphinx Wagtail theme](https://sphinx-wagtail-theme.netlify.app/)
+
+[![CI status](https://github.com/wagtail/sphinx_wagtail_theme/workflows/CI/badge.svg)](https://github.com/wagtail/sphinx_wagtail_theme/actions?query=workflow%3ACI) [![Netlify Status](https://api.netlify.com/api/v1/badges/82ecbf01-a706-4e92-8457-8a8726ca2123/deploy-status)](https://app.netlify.com/sites/sphinx-wagtail-theme/deploys) [![PyPI version](https://badge.fury.io/py/sphinx-wagtail-theme.svg)](https://badge.fury.io/py/sphinx-wagtail-theme) [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
 
 Sphinx Wagtail theme contains all files required to build a Sphinx extension that provides the theme.
 
-* Free software: MIT license
-* Authors: See AUTHORS.rst
-* Repository: https://github.com/wagtail/sphinx_wagtail_theme
-* Documentation: See docs/ folder
+-   [MIT license](https://github.com/wagtail/sphinx_wagtail_theme/blob/main/LICENSE)
+-   [AUTHORS](https://github.com/wagtail/sphinx_wagtail_theme/blob/main/AUTHORS.rst)
+-   [Repository](https://github.com/wagtail/sphinx_wagtail_theme)
+-   [Documentation](https://sphinx-wagtail-theme.netlify.app/)
+-   [Security](https://github.com/wagtail/sphinx_wagtail_theme/blob/main/SECURITY.md)
+-   Supports Python > 3.7
+
+## Installation
+
+```sh
+pip install sphinx-wagtail-theme
+```
+
+## Usage
+
+Select the "Sphinx Wagtail theme" in the `conf.py` file of a Sphinx
+
+```python
+# include the theme in the list of extensions to be loaded
+extensions = ['sphinx_wagtail_theme', …]
 
+# select the theme
+html_theme = 'sphinx_wagtail_theme'
+```
 
-## Release process
+-   See the documentation for more usage instructions
+
+## Development
+
+### Getting started
+
+-   Clone this repo locally
+-   Set up a Python virtual environment (e.g. `venv`)
+-   Show all available tasks via `make`
+-   Run `make setup` and then `make docs` to install and build the docs locally
+-   See the documentation for more development instructions
+
+### Release process
 
 Checklist:
 
-- AUTHORS.rst is updated
-- HISTORY.rst (change log) is updated
-- Everything is committed, clean checkout
-- ~/.pypirc has a username and password (token)
+-   AUTHORS.rst is updated
+-   HISTORY.rst (change log) is updated
+-   Everything is committed, clean checkout
+-   ~/.pypirc has a username and password (token)
+-   Add a git tag and a Github release once completed
 
 With an active virtual environment:
 
-    python -m pip install --upgrade -r requirements.txt
-    make clean
-    make clean-frontend
-    npm install
-    npm run build
-    prerelease
-    git tag -a N.N.N -m "N.N.N"                                                                                       ✘ PIPE  sphinx_wagtail_theme  14.16.0 23:21:46
```

Comment:
Thanks for cleaning up that weird terminal error in there 👍

Reply:
yeah, at first I thought it was intentional haha and then read again and realised it must have been a mistake